react-native-camera.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.source         = { :git => 'https://github.com/lwansbrough/react-native-camera', :tag => s.version }
+  s.source         = { :git => 'https://github.com/lwansbrough/react-native-camera', :tag => "v#{s.version}" }
```

Comment:
this should change: https://github.com/react-native-community/react-native-camera

Reply:
@ammer do you wanna update this?

Reply:
updated this in PR https://github.com/react-native-community/react-native-camera/pull/1256